tubuddy_web/api/forms.py

```diff
@@ -20,7 +20,7 @@ class RegisterForm(forms.Form):
     name = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Name', max_length=100)
     password = forms.CharField(widget=forms.PasswordInput(attrs={'class': 'form-control'}), label='Password',
                                max_length=100)
-    api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=100)
+    # api_token = forms.CharField(widget=forms.TextInput(attrs={'class': 'form-control'}), label='Api_token', max_length=100)
```

Comment:
Same for this?

Reply:
not that I know of

Reply:
Oh nevermind, I thought this was in our code.

'not our problem' ;)